scripts/vim-patch.sh

```diff
@@ -280,7 +280,7 @@ submit_pr() {
     echo "Pushing to 'origin/${checked_out_branch}'."
```

Comment:
1. What about specifying the remote to use optionally after `-s`?
I am using "origin" for neovim/neovim and cannot push there.
I am using "hub", and have not looked into using git-hub instead, which appears to not require this step?!

2. I have a function to push a branch and create a PR already, but that also handles the branch's upstream, which gets configured to be `origin/master` when using this script.
What about using `--no-track` with the `git checkout -b` for creating the branch?